pkg/garbagecollection/gc.go

```diff
@@ -158,3 +161,28 @@ func (gc *GC) collectDeployment(option metav1.ListOptions, runningSet map[types.
 
 	return nil
 }
+
+func (gc *GC) collectPVCs(option metav1.ListOptions, runningSet map[types.UID]bool) error {
```

Comment:
@sgotti 

when this lands, we probably already move to k8s 1.8, which supports crd GC natively. so we do not need this thing anymore.

/cc @hongchaodeng

Reply:
+1
We will `rm -rf` entire custom GC code :)

Reply:
done